StatusValidator.py

```diff
@@ -5,3 +5,59 @@
 class StatusValidator:
     def __init__(self):
         pass
+
+    def isWin(board: Board, player: Player, row: int) -> bool:
+        horizontalWin = StatusValidator.horizontalWin(board, player)
+        verticalWin = StatusValidator.verticalWin(board, player, row)
```

Comment:
The third parameter here is "row". In the function itself it is called column. Is this correct?